_pages/ato.md

```diff
@@ -14,13 +14,19 @@ While you’re in the ATO process, the following things will occur. These are la
 1. Once the SSP is complete, the system will be [scanned](../security/scanning/) in several ways.
 1. Information Security finalizes the SSP
 1. If all of this is completed, GSA Information Security signs and grants an ATO for a [one year term](types/#year-authorization).
-1. If the system substantively changes, then a new ATO will be warranted. The 18F DevOps team will make this determination – please open a new issue if you think you have made a change that may warrant a new ATO. Examples for triggers for a new ATO include changes to:
-    * Encryption methodologies
-    * A web framework having new backend administrative frameworks
-    * The kinds of information you begin to store (e.g. personally identifiable information)
 
 See [the checklist](checklist/) to see these tasks broken down.
 
 ### Signing an authorization
 
 Once the entire process is complete, GSA InfoSec will make a recommendation to the head of the GSA Office of Citizen Services and Innovative Technologies (OCSIT), our current Authorizing Official, for signature.
+
+### Re-authorization
+
+If the system substantively changes, then a new ATO will be warranted. Examples for triggers for a new ATO include changes to:
+
+* Encryption methodologies
+* Administrative functionality within the application
+* The kinds of information you begin to store (e.g. [personally identifiable information](../security/pii/))
+
+The 18F DevOps team will make this determination – please open a new issue if you think you have made a change that may warrant a new ATO.
```

Comment:
You mentioned changing all of those in https://github.com/18F/before-you-ship/pull/91, so I resisted worrying about it :smile: